libavcodec/x86/vvc/vvc_lmcs.asm

```diff
@@ -0,0 +1,255 @@
+%include "libavutil/x86/x86util.asm"
+
+SECTION_RODATA 32
+
+pb_shufmask_16bpc:   db 0, 1, 4, 5, 8, 9, 12, 13, -1, -1, -1, -1, -1, -1, -1, -1, 0, 1, 4, 5, 8, 9, 12, 13, -1, -1, -1, -1, -1, -1, -1, -1
+pb_shufmask_8bpc:  db 0, 4, 8, 12, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, 0, 4, 8, 12, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1
+
+SECTION .text
+
+; Only handles widths 16 to 128
+INIT_YMM avx2

```

Comment:
Better to follow the FFmpeg style. Like

#if HAVE_AVX2_EXTERNAL
INIT_YMM avx2
LMCS_16BPC
#endif

You can check the vvc_mc.asm for example.